app/controllers/comments_controller.rb

```diff
@@ -8,6 +8,7 @@ def new
 
   def create
     @user = current_user
```

Comment:
- After properly installing devise, please ensure that you remove the `current_user` method in your application controller as devise has a helper method for this instead. Please fix this 🙏